.github/workflows/mta-v7.x-e2e.yaml

```diff
@@ -115,18 +115,18 @@ jobs:
       - name: Deploy MTA serverless workflow
         run: |
           # Set the endpoint to the tackle-ui service
-          yq --inplace '.spec.podTemplate.container.env |= ( . + [{"name": "QUARKUS_REST_CLIENT_MTA_JSON_URL", "value": "http://tackle-ui.my-konveyor-operator.svc:8080"}, {"name": "BACKSTAGE_NOTIFICATIONS_URL", "value": "http://orchestrator-backstage.default.svc.cluster.local:7007/"}] )' manifests/01-sonataflow_mta-analysis-v7.yaml
+          yq --inplace '.spec.podTemplate.container.env |= ( . + [{"name": "QUARKUS_REST_CLIENT_MTA_JSON_URL", "value": "http://tackle-ui.my-konveyor-operator.svc:8080"}, {"name": "BACKSTAGE_NOTIFICATIONS_URL", "value": "http://orchestrator-backstage.default.svc.cluster.local:7007/"}] )' manifests/04-sonataflow_mta-analysis-v7.yaml
           
           # Disable persistence for e2e tests
-          yq e '.spec.persistence = {}' -i manifests/01-sonataflow_mta-analysis-v7.yaml
+          yq e '.spec.persistence = {}' -i manifests/04-sonataflow_mta-analysis-v7.yaml
           sed -i '/quarkus\.flyway\.migrate-at-start=true/d' manifests/03-configmap_mta-analysis-v7-props.yaml
 
           echo "manifests/03-configmap_mta-analysis-v7-props.yaml"
```

Comment:
pls change this to `01-...`

Reply:
sorry, did that already

Reply:
test still fails, probably misses the patching of the secret. not going to do it now. merging.

Reply:
Not sure about that, the MTA V7 e2e tests are failing for quite some time I think https://github.com/parodos-dev/serverless-workflows/actions/workflows/mta-v7.x-e2e.yaml

Reply:
so probably more work is required. one day...